pkg/plugins/platform/utils.go

```diff
@@ -45,9 +45,17 @@ func GetApps() map[string][]App {
 				}
 
 				err = site.Init(approot)
-				if err == nil {
-					apps[platformType] = append(apps[platformType], site)
+				if err != nil {
+					// Cast 'site' from type App to type LocalApp, so we can manually enter AppConfig values.
+					siteStruct, ok := site.(*LocalApp)
+					if !ok {
+						log.Fatalf("Failed to init %v err=%v", approot, err)
```

Comment:
I don't mind the path you took here, but the error message makes no sense. Please make it make sense. It needs to be about the failure of the cast.

Reply:
In this context I feel we would want this error to be the one presented. It is more important than the failure to cast because casting only happens when we can't find a config in the app's root directory. If we can't resolve the missing directory issue by casting, we want to return the overall error that the cast is nested in.

Reply:
I think presenting the failure to cast message on the line before the `log.Fatalf` would be a good compromise.